config.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+versions:
+  - name: '22.05'
+    modules: []
+  - name: dev
+    modules:
+      - name: openwisp-firmware-upgrader
+        branch: reorder-docs
+      - name: openwisp-controller
+        branch: reorder-docs
+      - name: openwisp-monitoring
+        branch: reorder-docs
+      - name: openwisp-notifications
+        branch: reorder-docs
+      - name: openwisp-network-topology
+        branch: reorder-docs
+      - name: openwisp-users
+        branch: reorder-docs
+      - name: openwisp-utils
+        branch: reorder-docs
```

Comment:
Can we change the implementation so that the order of this list is respected and the index here reflects this?

![Screenshot from 2024-04-29 21-26-20](https://github.com/openwisp/openwisp2-docs/assets/841044/be2fb37c-3e34-4016-ab8a-7f0dad64080f)

Without this change I am not sure we have a way to decide the order of these documents, which is problematic.

Reply:
Instead of using `modules/*/docs/*` in the TOC tree. We can specify the path of each module to get the desired order.